processing/src/main/java/org/apache/carbondata/processing/loading/parser/impl/MapParserImpl.java

```diff
@@ -44,12 +44,13 @@ public MapParserImpl(String delimiter, String nullFormat, String keyValueDelimit
         String[] split = pattern.split(value, -1);
         if (ArrayUtils.isNotEmpty(split)) {
           ArrayList<Object> array = new ArrayList<>();
-          Set<Object> set = new HashSet<>();
+          Map<Object, String> map = new HashMap<>();
```

Comment:
Change the comment at top of this method to describe this logic

Reply:
Added comment.